terraform/modules/ops/variables.tf

```diff
@@ -32,3 +32,9 @@ variable "setup_cd_system" {
   default     = false
   description = "Create deployment triggers. Enable only if Cloud Build has been granted GitHub access."
 }
+
+variable "setup_e2e_tests" {
```

Comment:
Sorry I missed this - this variable should be declared only in the `terraform/environments/ops/variables.tf` file

Reply:
Done.